sympy/sets/sets.py

```diff
@@ -1827,7 +1827,7 @@ def imageset(*args):
     return ImageSet(f, *set_list)
 
 
-def is_function_invertible_in_set(func, setv):
+def is_function_invertible_in_set(func, sym, setv):
```

Comment:
What's the purpose of a new variable?

Reply:
Actually, the earlier code was giving errors and therefore I was bound to take symbol as an argument.
I tried to implement the method on my system but it was giving errors and wrong outputs. Therefore, I need to modify the method.
I thought that, by taking symbol as an argument would keep all the methods uniform because every method takes symbols and functions as separate arguments.